mbf_costmap_nav/CMakeLists.txt

```diff
@@ -3,18 +3,18 @@ project(mbf_costmap_nav)
 
 find_package(catkin REQUIRED
   COMPONENTS
-  base_local_planner
   dynamic_reconfigure
   geometry_msgs
   mbf_abstract_nav
   mbf_costmap_core
-  mbf_msgs
   mbf_utility
+  mbf_msgs
```

Comment:
?

Reply:
it was not sorted... so I just grouped by loose groups